src/modules/FFmpeg/FFDecSW.cpp

```diff
@@ -339,8 +339,6 @@ bool FFDecSW::open(StreamInfo &streamInfo, VideoWriter *)
 		return false;
 	if (codec_ctx->codec_type == AVMEDIA_TYPE_VIDEO)
 	{
-		if (codec->capabilities & CODEC_CAP_DR1)
```

Comment:
> Maybe I'm missing something?

Yes, there is at lease one difference between FFmpeg 2.2 and 2.3 in usage of this flag, see: https://github.com/zaps166/QMPlay2/commit/d657f5ca409a621871e762c9f10e53f0de3659fa